src/actions/issue.js

```diff
@@ -4,27 +4,50 @@ import * as types from '../constants/actionTypes';
 import firebase from 'firebase';
 
 
+function _pluckIssueProps(array) {
+  const data = Object.keys(array).map(el => {
+    return {
+      id: array[el].number,
+      title: array[el].title,
+      body: array[el].body,
+      url: array[el].html_url,
+      votes: 0,
+      };
+    }
+  );
+  return data;
+}
+
 export function fetchIssuesFromGH() {
   return dispatch => {
     dispatch({
       type: types.REQUEST_ISSUES_GH,
     });
     return fetch('https://api.github.com/repos/zooniverse/wildcam-gorongosa-education/issues?labels=education-api')
       .then(response => response.json())
-      .then(array => dispatch({
-        type: types.RECEIVE_ISSUES_SUCCESS_GH,
-        data: array,
-        error: false,
-        issue: null,
-        loading: false,
-      }))
+      .then(array => {
```

Comment:
no need for curly braces, you can pass the dispatch function as you do in the catch block below